packages/esm-patient-search-app/src/patient-search-page/refine-search/person-attribute-field.component.tsx

```diff
@@ -0,0 +1,232 @@
+import React, { useCallback, useMemo, useRef, useState } from 'react';
+import { ComboBox, InlineLoading, InlineNotification, TextInput, TextInputSkeleton } from '@carbon/react';
+import { useTranslation } from 'react-i18next';
+import { useAttributeConceptAnswers, useLocations, usePersonAttributeType } from './person-attributes.resource';
+import { type AdvancedPatientSearchState, type SearchFieldConfig } from '../../types';
+import styles from './search-field.scss';
+
+export interface PersonAttributeFieldProps {
+  field: SearchFieldConfig;
+  formState: AdvancedPatientSearchState;
+  inTabletOrOverlay: boolean;
+  isTablet: boolean;
+  onInputChange: (fieldName: string) => (evt: { target: { value: string } } | { name: string }) => void;
```

Comment:
```suggestion
  onInputChange: (fieldName: string) => (evt: { target: { value: string } } | { value: string }) => void;
```

Also, the key in the second object should be `value` instead of `name`.